nais/topics.yml

```diff
@@ -0,0 +1,69 @@
+apiVersion: kafka.nais.io/v1
+kind: Topic
+metadata:
+  name: privat-omsorgsdager-melding-mottatt
+  namespace: dusseldorf
+  labels:
+    team: dusseldorf
+spec:
+  pool: {{kafka-pool}}
+  config: # optional; all fields are optional too; defaults shown
+    cleanupPolicy: delete  # delete, compact
+    minimumInSyncReplicas: 1
+    partitions: 1
+    replication: 3  # see min/max requirements
+    retentionBytes: -1  # -1 means unlimited
+    retentionHours: 730  # -1 means unlimited
+  acl:
+    - team: dusseldorf
+      application: omsorgsdager-melding--api
```

Comment:
Det er to bindesteker (--) mellom melding og api.

Reply:
Jaggu! Rart det gikk likevel

Reply:
Hmm snodig :P 
Bør fikses likevel da :)

Reply:
Ja absolutt